components/powerMgr/linux/le_ulpm.c

```diff
@@ -497,15 +498,20 @@ le_result_t le_ulpm_ShutDown
     if (pm_CheckWakeLock())
     {
         LE_ERROR("Wakelock held!! System can't be shut down. Try again.");
-        return LE_UNAVAILABLE;
+        return LE_NOT_POSSIBLE;
```

Comment:
This value is not declared as a return value for le_ulpm_ShutDown.
Also that would modify an existing API, so it might be best to keep it as this.